packages/request/package.json

```diff
@@ -32,6 +33,7 @@
     "@typescript-eslint/eslint-plugin": "^5.62.0",
     "@typescript-eslint/parser": "^5.62.0",
     "chai": "^4.3.8",
+    "dotenv": "^16.3.1",
```

Comment:
Not a good idea to add a new dev dependency

Reply:
I had to add it to be able to use environmental variables.

Reply:
Reverted.